libraries/lib-screen-geometry/ViewInfo.cpp

```diff
@@ -146,23 +134,10 @@ bool NotifyingSelectedRegion::setF1(double f, bool maySwap)
 
 void NotifyingSelectedRegion::Notify( bool delayed )
 {
-   SelectedRegionEvent evt{ EVT_SELECTED_REGION_CHANGE, this };
-   if ( delayed )
-      QueueEvent( evt.Clone() );
+   if (delayed)
+      BasicUI::CallAfter([this]{ Publish({}); });
```

Comment:
Same question here: is it guaranteed that the `NotifyingSelectedRegion` instance exists when lambda is called?

Reply:
I feel less confident.  I'll rewrite with a weak pointer.